build.xml

```diff
@@ -245,27 +245,22 @@
         <string>--add-exports java.base/java.lang.ref=ALL-UNNAMED</string>
         <string>--add-exports jdk.unsupported/sun.misc=ALL-UNNAMED</string>
         <string>--add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED</string>
+        <string>--add-exports jdk.attach/sun.tools.attach=ALL-UNNAMED</string>
 
         <string>--add-opens java.base/java.lang.module=ALL-UNNAMED</string>
-        <string>--add-opens java.base/java.net=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.loader=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.ref=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.reflect=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.math=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.module=ALL-UNNAMED</string>
         <string>--add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED</string>
         <string>--add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED</string>
-        <string>--add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED</string>
-
         <string>--add-opens java.base/sun.nio.ch=ALL-UNNAMED</string>
```

Comment:
If we have an `--add-opens` for `java.base/sun.nio.ch` do we need the `--add-exports` ?
Same question for` java.base/java.lang`

Reply:
I see only `--add-opens java.base/java.lang=ALL-UNNAMED`. There is `--add-exports java.base/java.lang.ref=ALL-UNNAMED`, but that one is different.
Valid point for` java.base/sun.nio.ch`. My understanding is yes, but I do not find an affirmative java doc on that so I might prefer the redundancy. The add-exports was added back in C* 4.0. WDYT?

Reply:
Could we try to remove it?